managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -305,8 +304,6 @@ public ManagedLedgerImpl(ManagedLedgerFactoryImpl factory, BookKeeper bookKeeper
         this.uninitializedCursors = Maps.newHashMap();
         this.clock = config.getClock();
 
-        // Get the next rollover time. Add a random value upto 5% to avoid rollover multiple ledgers at the same time
-        this.maximumRolloverTimeMs = getMaximumRolloverTimeMs(config);
```

Comment:
Looks like it will never change since it creates. Shouldn't we change the place which uses `getMaximumRolloverTimeMs` to the `maximumRolloverTimeMs` to avoid unnecessary compute?

Reply:
Yes, I think we should use the `maximumRolloverTimeMs`, and should avoid the `updateLastLedgerCreatedTimeAndScheduleRolloverTask` method to compute the `maximumRolloverTimeMs` for every check.

Reply:
Yes. you are right. Fixed